applications/FemToDemApplication/python_scripts/check_and_prepare_model_process.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+#import kratos core and applications
+import KratosMultiphysics
+import KratosMultiphysics.SolidMechanicsApplication as KratosSolid
+
+def Factory(settings, Model):

```

Comment:
With the new Model many of this tings are outdated. Check the StructuralMechanicsApplication